internal/helpers/prometheus_helper.go

```diff
@@ -118,7 +118,7 @@ func (mrs *MonitoringRuleSet) createSliMeasurementRecordingRule(totalRule, goodR
 	totalLabels := mapToColonSeparatedString(totalRule.Labels)
 	return monitoringv1.Rule{
 		Record: fmt.Sprintf("%s_sli_measurement", RecordPrefix),
-		Expr:   intstr.FromString(fmt.Sprintf("%s{%s} / %s{%s}", goodRule.Record, goodLabels, totalRule.Record, totalLabels)),
+		Expr:   intstr.FromString(fmt.Sprintf("clamp_max(%s{%s} / %s{%s})", goodRule.Record, goodLabels, totalRule.Record, totalLabels)),
```

Comment:
```suggestion
		Expr:   intstr.FromString(fmt.Sprintf("clamp_max(%s{%s} / %s{%s}, 1)", goodRule.Record, goodLabels, totalRule.Record, totalLabels)),
```

Clamp max needs a second argument, doesn't it?

Reply:
Yeah... I tested it in Grafana after I did this, fixed it in the Grafana query but never returned to fix it in the code. 

Good review, thx :D